WORKSPACE

```diff
@@ -0,0 +1,64 @@
+workspace(
+    name = "cdk8s_srcs",
+    managed_directories = {"@npm": ["node_modules"]},
+)
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
what is skylib? add a comment explaining why is it needed?

Reply:
This is, in fact, not needed. At one point we depended on it for the `bzl_library` rule, but I got rid of that in one of the drafts. Will remove in the next commit.

Reply:
Redacted above. We do need this for `copy_bash` used in the `jsii_library`. I'll add a comment.

Reply:
Wouldn't it make sense to load it as part of the definition of `jsii_library` instead of at the workspace level?

Reply:
It's loaded there, but it's _downloaded_ here. That's how Bazel works, all necessary tools/libraries are fetched and setup in the WORKSPACE file, then consumed in the BUILD/*.bzl files.